test/test_regression.py

```diff
@@ -167,8 +167,9 @@ def test_csv_regression(dataset, expected, tmpdir):
 
 
 @pytest.mark.parametrize("dataset, expected", download_md5)
-def test_download_regression(dataset, expected):
+def test_download_regression(dataset, expected, tmpdir):
```

Comment:
I think we don't need` , tmpdir`

Reply:
This one always confuses me as well. I get errors without it and believe it's part of how the tmpdir system works in pytest.